### DISCUSSION TITLE
Add support for long-term statistics sensors

### DISCUSSION THREAD

**Discussion by sefininio** [zachowj/node-red-contrib-home-assistant-websocket#466] [Feature Request]
The `entity` node allows for creating `binary_sensor`, `sensor` and `switch` entities on the HomeAssistant instance via the `Node-RED Companion`.

HomeAssistant recently introduced [Long-Term statistics](https://developers.home-assistant.io/docs/core/entity/sensor/#long-term-statistics) sensors - those are needed, for example, to create sensors compatible with the new energy dashboard.

**Comment by sefininio** [comment]
I think it basically means adding the `state_class` prop to the `home assistant config` section of the node.

**Reply by zachowj** [reply]
https://github.com/zachowj/node-red-contrib-home-assistant-websocket/discussions/459#discussioncomment-1195358

**Reply by sefininio** [reply]
Oh sorry, I missed it's a dup.
Well, I tried adding the attribute and that works! Thanks!

**Comment by zachowj** [comment]
Added in v0.40.0
